custom_components/alexa_media/alexa_entity.py

```diff
@@ -41,6 +41,16 @@ def has_capability(
     return False
 
 
+def is_emulated_hue(appliance: Dict[Text, Any]) -> bool:
+    """Determine if an appliance may have originated from an emulated hue hub.
+
+    This check will catch all emulated_hue devices, but it may also catch too much.
+    Its possible some older Hue bulbs may actually use this manufacturer name even though modern bulbs report "Philips".
+    Therefore, only believe this check if emulated_hue is actually turned on.
+    """
+    return appliance.get("manufacturerName") == "Royal Philips Electronics"
```

Comment:
Shouldn't the check for emulated_hue be also here? Otherwise this function seems incomplete since it really isn't answering the question.

Reply:
I left the core logic and checks the same but renamed the function, property, and rewrote the comments. Hopefully, that makes the code much more obvious. My thinking is that this code is mostly concerned with parsing and the policy decision on what to do with old Hue stuff should be made by the platform itself.